sphinx/source/docs/dev_guide/testing.rst

```diff
@@ -355,7 +351,7 @@ separate ``test:<GROUP>`` script for each of the following test groups:
     a report is uploaded to see the detailed results.
 
 ``js``
-    This job runs all the JavaScript unit tests (i.e. ``gulp test``)
+    This job runs all the JavaScript unit tests (i.e. ``node make test``)
 
```

Comment:
I think the setup section of the dev guide also references gulp

Reply:
I thought there was mention of it in the install section, but apparently I was mistaken.